Code/Framework/AzFramework/AzFramework/Components/CameraBus.h

```diff
@@ -129,6 +129,32 @@ namespace Camera
                 GetFrustumHeight()
             };
         }
+
+        //! Unprojects a position in screen space pixel coordinates to world space.
+        //! With a depth of zero, the position returned will be on the near clip plane of the camera
+        //! in world space.
+        //! @param screenPosition The absolute screen position
+        //! @param depth The depth offset into the world relative to the near clip plane of the camera 
+        //! @return the position in world space
+        virtual AZ::Vector3 ScreenToWorld(const AZ::Vector2& screenPosition, float depth) = 0;
+
+        //! Unprojects a position in screen space normalized device coordinates to world space.
+        //! With a depth of zero, the position returned will be on the near clip plane of the camera
+        //! in world space.
+        //! @param screenNdcPosition The normalized device coordinates in the range [0,1]
+        //! @param depth The depth offset into the world relative to the near clip plane of the camera 
```

Comment:
suggestion: It might be worth while to be explicit that `depth` is in worldSpace and not in the `[0-1]` range